src/shared/utils/radarUtils.ts

```diff
@@ -0,0 +1,143 @@
+import * as d3 from 'd3';
+import * as R from 'ramda';
+import { color } from '../../theme';
+
+// custom random number generator, to make random sequence reproducible
+// source: https://stackoverflow.com/questions/521295
+let seed = 42;
+export const random = () => {
+  const x = Math.sin(seed++) * 10000;
+  return x - Math.floor(x);
+};
+
+export const random_between = (min: number, max: number) => {
```

Comment:
types for `min: number, max:number` are repeating
we could use interface for this.